src/cli/OrganizationsCLIController.ts

```diff
@@ -13,30 +13,58 @@ export type Organization = {
 
 export class OrganizationsCLIController extends BaseCLIController {
   projectController: ProjectsCLIController
+  baseCLIController: BaseCLIController
```

Comment:
the OrganizationsCLIController is extending the BaseCLIController, so it shouldn't need to create an instance of it as well